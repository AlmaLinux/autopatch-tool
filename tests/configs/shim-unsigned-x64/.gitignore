SOURCES/redhatsecurebootca5.cer
SOURCES/shim-15.8.tar.bz2
