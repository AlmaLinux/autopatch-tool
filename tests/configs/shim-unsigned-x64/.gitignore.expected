SOURCES/shim-15.8.tar.bz2
